yarn.lock

```diff
@@ -12112,6 +12112,11 @@
     "react-fast-compare" "^3.2.0"
     "shallowequal" "^1.1.0"
 
+"react-icons@^4.4.0":
```

Comment:
It is a bit weird to have both package-lock.json and yarn.lock... Normally you have only one of them to manage your packages (either use of npm install or yarn install).